tensorboard/components/tf_markdown_view/tf-markdown-view.html

```diff
@@ -0,0 +1,82 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+  tf-markdown-view renders raw HTML that has been converted from
+  Markdown by some other agent. The HTML must be sanitized, and must be
+  safe to inject directly into the DOM.
+
+  TensorBoard plugins can use the `markdown_to_safe_html` function from
+  the `tensorboard.plugin_util` module on the backend to generate HTML
+  suitable for use with this component.
+-->
+<dom-module id="tf-markdown-view">
+  <template>
+    <div id="markdown" inner-h-t-m-l="[[html]]"></div>
+    <style>
+      /*
+       * Reduce topmost and bottommost margins from 16px to 0.3em (renders
```

Comment:
nit: This comment was there before, so this is a bit irrelevant to this change, but 0.3em actually varies in pixels based on the font used by the parent component. It's probably generally accurate, although maybe '5px' suffices. 4.8 seems oddly precise. :)